src/server/tests/config_test.py

```diff
@@ -0,0 +1,40 @@
+from server.config import get_config, get_chapters, get_languages, get_live, \
+    SUPPORTED_YEARS, SUPPORTED_CHAPTERS, SUPPORTED_LANGUAGES
+
+
+test_year = ['2020', '2019', '2018']
+
+
+def test_get_config_year():
+    assert get_config(test_year[0]) is not None
+    assert get_config(test_year[1]) is not None
+    assert get_config(test_year[2]) is None
+
+
+def test_static_avatar():
+    for contributor in get_config('2020'):
+        if contributor == 'michelleoconnor':
+            static_url = contributor['avatar_url']
+            assert static_url is not None and static_url.startswith('/static')
```

Comment:
Let's add a comment here to explain why we're hard coding a users:

```suggestion
def test_static_avatar():
    # For this test we use an existing contributor with no avatar as they should be assigned one on initial load
    for contributor in get_config('2020'):
        if contributor == 'michelleoconnor':
            static_url = contributor['avatar_url']
            assert static_url is not None and static_url.startswith('/static')
```